app/src/main/java/com/esri/arcgismaps/kotlin/sampleviewer/ui/screens/home/HomeCategoryScreen.kt

```diff
@@ -116,10 +119,25 @@ fun HomeCategoryScreen(
                     CategoryCard(category) { onNavigateToCategory(category.title) }
                 }
             }
+            CheckAccessToken()
         }
     }
 }
 
+@Composable
+fun CheckAccessToken() {
+    var isDialogDisplayed by remember {
+        mutableStateOf(BuildConfig.ACCESS_TOKEN == "DEFAULT_ACCESS_TOKEN")
+    }
+    if (isDialogDisplayed) {
+        MessageDialog(
+            title = "Using default access token",
+            description = "Add an ACCESS_TOKEN to the project's local.properties",
```

Comment:
Do we want to include a hyperlink to the guide doc here?

Reply:
I cannot add a link to the string params of the `MessageDialog` instead, I added a link to repo instructions on setting the key as part of the function comment.